src/app/components/layout/navbar/navbar.component.ts

```diff
@@ -167,22 +175,10 @@ export class NavbarComponent implements OnInit, AfterViewInit, OnDestroy {
           northboundServices.forEach(service => {
             this.servicesRecord.push(service);
           });
-          if (notificationService.length) {
-            this.servicesRecord.push(notificationService[0]);
-          }
-
-          if (managementService.length) {
-            this.servicesRecord.push(managementService[0]);
-          }
-
-          if (dispatcherService.length) {
-            this.servicesRecord.push(dispatcherService[0]);
-          }
```

Comment:
Did we test vanilla and adding one/few of them only; in only installed vs disabled/enabled added state?